content/2. Ondersteunende-informatie/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/1. If-Else.md

```diff
@@ -0,0 +1,98 @@
+
+## Hoe zit een If-Else-structuur in elkaar?
+De *If-Else*-structuur volgt een specifieke opbouw, die begint met een `if`-clausule waarin een conditie wordt gespecificeerd, gevolgd door de bijbehorende actie. Optioneel volgt daarna een `else`-clausule waarin een alternatieve actie kan worden gedefinieerd. 
+
+Bijvoorbeeld:
+```C#
+// Optie voor if-else
+if (conditie_1){
+	// code om uit te voeren als de conditie waar is.
+}
+else{
+    // alternatieve code als de conditie niet waar is.
+}
+
+// Optie voor alleen een if
+if (conditie_2){
+	// code om uit te voeren als de conditie waar is.
+}
+```
+  Dit model toont de basisopbouw van de *If-Else*-logica.
+
+## Hoe gebruik je een If-Else-structuur?
+De causaliteit in een *If-Else*-structuur komt voort uit de vraag: "Wat gebeurt er als een conditie al dan niet waar is?" Het gebruik van *If-Else* stelt het programma in staat om gedrag te vertonen op basis van bepaalde omstandigheden, wat leidt tot verschillende uitkomsten. Bijvoorbeeld, als de gebruiker ouder is dan 18, krijgt hij/zij toegang, anders wordt de toegang geweigerd.
+
+Stel, een schoolapplicatie gebruikt een *If-Else*-structuur om te bepalen of een student toegang krijgt tot bepaalde educatieve content. Als de student meerderjarig is moet hij of zij toegang krijgen. Als de student minderjarig is krijgt hij of zij geen toegang.
+
+In het onderstaande voorbeeld is een variabel `age` aanwezig. Dit is de leeftijd van de student.
+Als de waarde van het variabel `age` groter of gelijk is aan `18` moet de student toegang krijgen.
+Als de waarde van het variabel `age` niet voldoet aan de conditie, dan gaat het programma naar de `else` statement.
+```C#
+int age = 16;
+if (age >= 18){
+    Console.WriteLine("Toegang verleend tot de content.");
+}
+else {
+    Console.WriteLine("Toegang geweigerd. Leeftijdsrestrictie: 18+");
+}
+```
+
+## Wat is een If-Else-If-structuur?
+Een *If-Else-If* structuur biedt een uitbreiding op de eenvoudige *If-Else*-structuur. Het maakt het mogelijk om meerdere condities achter elkaar te controleren. Wanneer een conditie waar (true) is, voert het programma de corresponderende code uit en negeert het de overige condities. Dit stelt de programmeur in staat om een serie aan beslissingen in een logische volgorde te plaatsen zonder herhaald gebruik van geneste *If-Else*-structuren, waardoor de code overzichtelijker en beter leesbaar blijft.
+
+## Hoe zit een If-Else-If-structuur in elkaar?
+- Het programma controleert de eerste conditie in de `if`-clausule.
+- Als de conditie niet waar is, gaat het programma verder naar de volgende conditie in de `else if`-clausule.
+- Als de `else if`-clausule ook niet waar is, dan wordt de `else`-clausule bereikt indien deze aanwezig is. Deze fungeert hier als een vangnet voor de overige gevallen.
```

Comment:
Suggestie: else wordt pas bereikt wanneer er ook geen andere else-if aanwezig is (zoals je later laat zien dat dit wel degelijk mogelijk is)